.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "eslint.codeActionsOnSave.mode": "all",
+    "eslint.enable": true,
+    "eslint.run": "onSave",
+    "typescript.format.insertSpaceAfterSemicolonInForStatements": true,
+    "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+        "source.organizeImports": "explicit",
+    },
+    "typescript.updateImportsOnFileMove.enabled": "always",
+  "eslint.validate": ["javascript", "typescript"],
+  "[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "typescript.preferences.quoteStyle": "double",
+  "typescript.format.semicolons": "insert",
+  "eslint.lintTask.enable": true,
+  "eslint.lintTask.options": "-c ./app/.eslintrc.cjs --ext .ts --parser-options project:'./app/tsconfig.json' ./app"
```

Comment:
I'm not sure this will lint the generator package. I know there's some option to tell VSCode about a npm workspace which might make it work for both packages.